hh_plot.py

```diff
@@ -19,18 +19,41 @@ def main():
     parser.add_argument(
         "-o", "--outfile", help="Filename for output plot", required=True, type=str
     )
+    parser.add_argument(
+        "-z",
+        "--z_height",
+        help="Filename for output plot",
+        required=False,
+        type=float,
+        default=90.0,
+    )
+    parser.add_argument(
```

Comment:
can you do that action store true or whatever that thing is? Might be simpler switch. unless this is the same?